.github/workflows/increment-version-on-merge.yml

```diff
@@ -43,8 +43,17 @@ jobs:
       # See https://github.com/im-open/git-version-lite for more details around how to increment
       # major/minor/patch through commit messages
       - name: Increment the version
-        uses: im-open/git-version-lite@main
+        uses: ./
+        id: version
         with:
           create-ref: true
           github-token: ${{ secrets.GITHUB_TOKEN }}
           default-release-type: major
+
+      - name: Create Major and Latest Tags
+        uses: im-open/create-tags-action@v1
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          sha: ${{ steps.version.outputs.NEXT_VERSION_SHA }}
```

Comment:
Pass in a SHA instead of using the current context to future proof the issue raised in #24

Reply:
Personally, I'd like to eliminate the use of the `create-ref` feature of `git-version-lite`.

So, I'd prefer this workflow to use `git-version-lite` _only_ to compute the new version. Then replace this "Create Major Tag" step with a step that just runs the `git` CLI to push (1) the full semver tag (`NEXT_VERSION`) and (2) the major version tag (`NEXT_MAJOR_VERSION`) based on `${{ github.sha }}`

```
git tag ${{ steps.version.outputs.NEXT_VERSION }} ${{ github.sha }}
git push origin ${{ steps.version.outputs.NEXT_VERSION }}
git tag ${{ steps.version.outputs.NEXT_MAJOR_VERSION }} ${{ github.sha }}
git push origin ${{ steps.version.outputs.NEXT_MAJOR_VERSION }} -f
```

Of course using the git CLI will require a couple `git config` calls to go along with the checkout.

Reply:
If I recall I attempted this approach first and the workflow permissions denied the ability to push force the tag.  Why I instead used the GitHub Octokit to do the same.

But I can try it again in a separate PR again to verify.

Reply:
It actually might of not worked for me because I was on a fork.

Reply:
Attempt: #34